documentation/Set-PnPSiteDesign.md

```diff
@@ -141,7 +141,7 @@ Accept wildcard characters: False
 ```
 
 ### -SiteScriptIds
-An array of guids of site scripts
+An array of guid's of site scripts
```

Comment:
The previous version is the correct one.

Reply:
updated